packages/rum-core/test/performance-monitoring/performance-monitoring.spec.js

```diff
@@ -204,7 +205,32 @@ describe('PerformanceMonitoring', function() {
     expect(result).toBeDefined()
   })
 
-  it('should filter transactions', function() {
+  it('should filter transactions based on duration and spans', () => {
+    configService.setConfig({
+      transactionDurationThreshold: 200
+    })
+    spyOn(logger, 'debug').and.callThrough()
+    const transaction1 = new Transaction()
+    transaction1.end()
+    transaction1._start = 0
+    transaction1._end = 201
+    expect(transaction1.duration()).toBe(201)
+    expect(performanceMonitoring.filterTransaction(transaction1)).toBe(false)
+    expect(logger.debug).toHaveBeenCalledWith(
+      'Transaction was discarded! transaction duration: 201 contains falsy value or it is greater than transactionDurationThreshold: 200 config'
```

Comment:
> Transaction was discarded! transaction duration: 201 contains falsy value or it is greater than transactionDurationThreshold: 200 config

Is reads pretty awkwardly. I think it'd be easier to read as:

> Transaction was discarded! transaction duration (201) is greater than the transactionDurationThreshold configuration (200)

Reply:
Hmmm, can split those two if statements and can do this instead 👍

Reply:
Done